src/loaddb/load_error_handler.hpp

```diff
@@ -95,40 +98,52 @@ namespace cubload
   void
   error_handler::on_error (MSGCAT_LOADDB_MSG msg_id, Args &&... args)
   {
-    std::string err_msg = format (get_message_from_catalog (msg_id), std::forward<Args> (args)...);
-    log_error_message (err_msg, false);
+    if (!is_last_error_filtered ())
```

Comment:
I am not sure we need or can filter errors with MSGCAT_LOADDB_MSG. they don't seem to be filtered on SA_MODE either.

Probably filtering on_failure () with no arguments is enough? Please verify and confirm.